owl/EASE-PROC.owl

```diff
@@ -36,7 +36,7 @@
         <rdfs:comment>A relation between a description and a process type.</rdfs:comment>
         <rdfs:label>defines process</rdfs:label>
     </owl:ObjectProperty>
-    
+
```

Comment:
it might be that you have used a different Protege version for editing, which causes the diff to be bloated.

Potentially only way to avoid ping-pong'ing large diffs is to use exactly the same version. We haven't documented this, but it is certainly needed to avoid these formatting diffs

Reply:
I cleaned up the diffs. It was automatic trailing whitespace removal from another editor, mostly.